lib/utils/validate-global-prefix.util.ts

```diff
@@ -0,0 +1,2 @@
+export const validateGlobalPrefix = (globalPrefix: string): boolean =>
+  globalPrefix && !globalPrefix.match(/^(\/?)$/);
```

Comment:
```suggestion
  !globalPrefix.match(/^(\/?)$/);
```

also, can you add some jsdoc here to clarify _what_ this function is validating exactly?